lib/resources/EnterpriseZoneWorkersRoutes.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (C) 2014-present Cloudflare, Inc.
+
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+'use strict';
+
+const prototypal = require('es-class');
+const auto = require('autocreate');
+
+const Resource = require('../Resource');
+
+/**
+ * EnterpriseZoneWorkersRoutes represents the zones/:zoneId/workers/routes API endpoint.
+ *
+ * @class EnterpriseZoneWorkersRoutes
+ * @hideconstructor
+ * @extends Resource
+ */
+module.exports = auto(
+  prototypal({
+    extends: Resource,
+    path: 'zones/:zoneId/workers/routes',
+
+    includeBasic: ['browse', 'read', 'edit', 'add', 'del'],
+
+    /**
+     * browse allows for listing all of a zone's workers routes
+     *
+     * @function browse
+     * @memberof EnterpriseZoneWorkersRoutes
+     * @instance
+     * @async
+     * @param {string} zone_id - The zone ID
+     * @returns {Promise<Object>} The route browse response object.
+     */
+    /**
+     * read allows for retrieving a specific zone's workers route
```

Comment:
The documentation doesn't say you can get a specific route, so we might need to check this. @rita3ko

Reply:
Same. Just need to add to docs. Will do so shortly.